lib/chat_entry.dart

```diff
@@ -31,7 +31,8 @@ class _ChatEntryState extends State<ChatEntry> {
     _selectedName = widget.names.first;
     _names = ['Everyone', ...widget.names];
 
-    _controller = _ChatTextController(maxCharacters: _maxCharacters)..addListener(_onTextChanged);
+    _controller = _ChatTextController(maxCharacters: _maxCharacters)
```

Comment:
This because I have changed my line length, the one formatting option there is. A team should agree on this, otherwise changes like this can oscillate depending on who checked in last